.github/meta_issue_template/release-1.1.1-plugin-issues-min.md

```diff
@@ -0,0 +1,53 @@
+This is a component issue for release 1.1.1.
```

Comment:
What is the usage of this `-min` vs the other issue?

Reply:
It is only used for OpenSearch and Dashboards core repo.
Since they do not need any changes they also do not need any specific tests.

Reply:
Can we adapt the single template to cover this case in the with a "What to do if you have no changes section".  This second template requires more work by the release manager that it would be better to avoid.

Reply:
Fixed.

Reply:
```
- [ ] If including changes in this release, increment the version on 1.1 branch to `1.1.1` for Min/Core, and `1.1.1.0` for components. Otherwise, keep the version number unchanged for both.
```